wpilib/wpilib/i2c.py

```diff
@@ -35,11 +36,12 @@ class Port:
         kOnboard = 0
         kMXP = 1
 
-    def __init__(self, port, deviceAddress, simPort=None):
+    def __init__(
+        self, port: Port, deviceAddress: int, simPort: Optional[object] = None
```

Comment:
I'm not sure `object` is an appropriate type hint here. hm...

Reply:
I mean, the best thing would be an interface, but it's not even a full implementation of an interface, just the methods you use. Do you have any recommendations?

Reply:
Honestly, I would just leave that argument untyped. I don't see a nice way to do this.